lib/wickedbase.pm

```diff
@@ -138,8 +138,9 @@ sub get_ip {
         vlan_changed => ['192.0.2.111/24',           '192.0.2.110/24'],
         macvtap      => ['10.0.2.18/15',             '10.0.2.17/15'],
         bond         => ['10.0.2.18',                '10.0.2.17'],
-        dhcp_2nic    => ['10.0.3.15',                '10.0.3.16'],
+        dhcp_2nic    => ['10.20.30.',                '10.20.30.12'],
```

Comment:
`10.20.30.` looks wired

Reply:
it's because this IP is not fixed, and we can't know the last octet since it's given by DHPC.
Legacy suite does the same.
[See here](https://github.com/openSUSE/wicked-testsuite/blob/013874c1c4262f8b484837ee32e397205a087e60/features/step_definitions/wicked_definitions.rb#L48)

Reply:
Then I would add a comment and/or write it like `10.20.30.x`.

Reply:
`.x` does not help as I need to compare strings later on. I left a comment.